content/guidance/financial-support-for-teacher-training/_bursaries-cta.html.erb

```diff
@@ -1 +1,3 @@
-<%= render CallsToAction::ChatOnlineComponent.new %>
+<%= render CallsToAction::ChatOnlineComponent.new(
+    text: "If you're unsure about whether you could be entitled to a bursary or scholarship, you can chat to us.",
```

Comment:
Passing custom text into the `ChatOnlineComponent` isn't currently supported. I will create a PR to support this shortly.

https://github.com/DFE-Digital/get-into-teaching-app/blob/master/app/components/calls_to_action/chat_online_component.html.erb